content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,62 @@
+# Machine Learning Model Registry
+
+A **model registry** is a tool to catalog ML models and their versions. Models
+from your data science projects can be discovered, audited, shared, tested, and
+deployed from here. [Iterative](https://iterative.ai/) tools like
+[DVC](https://dvc.org/doc), [GTO], and [MLEM] enable these capabilities on top
+of Git, so you can stick to an existing software engineering stack. No more
+divide between ML engineering and operations!
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries unifies ML development and application. See also [Data Registries]._
+
+[gto]: https://github.com/iterative/gto
+[mlem]: https://mlem.ai/
+[data registries]: /doc/use-cases/data-registries
+
+Model registries can give your team key capabilities:
+
+- Collect and present your models, from early-stage [experiments] to
+  production-ready, including their [metrics, plots], or other metadata to help
+  use and evaluate them.
+- Organize and track model [versions] effectively; with full data provenance and
+  lineage information based on [reproducible modeling].
+- A single interface to access specific models, both for collaboration (from
+  other teammate machines) and to productionize them (e.g. via CI/CD).
+- Designate certain models for different environments stages (dev, shadow, prod,
+  etc.) without having to touch the applications that use them.
+- For security, control who can manage models, and audit their usage trails.
+
+[experiments]: /doc/user-guide/experiment-management
+[metrics, plots]: /doc/start/metrics-parameters-plots
+[versions]: /doc/use-cases/versioning-data-and-model-files
+[reproducible modeling]: /doc/start/data-pipelines
+
+Many of these benefits are built into DVC: Your [modeling process] and
+performance information become **codified** in Git-based <abbr>DVC
+projects</abbr>. This makes it possible to manage them with standard Git
+workflows along with code. Large model files are stored separately and
+efficiently, and can be pushed to [remote storage] -- a scalable access point
+for [sharing].
+
+An additional route to build a Git-native registry is to **annotate** ML models
+in your repos using tags. [GTO] (Git Tag Ops) is a tool to do just that! The
+annotations carry meaningful metadata, such as a model file path, deployment
+env, etc. and get codified in the repo. GTO also helps manage your models
+lifecycle: promoting versions, rolling back, and reviewing history.
```

Comment:
In fact, with git tags we register semantic versions and promote them to production. We `annotate` by writing to `artifacts.yaml`. Also, I would bring forward the use cases instead of providing some details, e.g.

```suggestion
An additional route to build a Git-native registry is to use git tags to **register**
and **promote** [semantic versions](https://semver.org) of ML models. 
[GTO] (Git Tag Ops) helps you do that. Besides helping you manage models lifecycle, 
GTO helps you **annotate** your models to define the registry-level names for your DVC outputs.
```

By `registry-level names` I mean human-friendly names you would like to see in MR, instead of some path. I'm sure there should be a better wording to explain this)

Reply:
Thanks! The suggested text is too technical, but I rewrote the GTO paragraph to mention the use cases more accurately. PTAL @aguschin 🙏🏼